console/console-init/ui/src/modules/iot-device/containers/DeviceListContainer/DeviceListContainer.tsx

```diff
@@ -72,7 +72,29 @@ export const DeviceListContainer: React.FC<IDeviceListContainerProps> = ({
 
   const { total, devices } = data?.devices || {};
```

Comment:
instead of empty object use this :- `const {total,devices}= data?.devices || {total:0,devices:[ ] }`